rust/shared/src/contract_utils/handler_result.rs

```diff
@@ -2,7 +2,9 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Serialize, Deserialize)]
 #[serde(untagged)]
-pub enum HandlerResult<State, QueryResponseMsg> {
+pub enum HandlerResult<State, ViewResponse> {
     NewState(State),
-    QueryResponse(QueryResponseMsg),
+    ViewResponse(ViewResponse),
 }
+
+pub type ActionResult<S, V, E> = Result<HandlerResult<S, V>, E>;
```

Comment:
can we use more descriptive names for the generic types?

so instead of S, V, E - STATE, VIEW, ERROR - or sth like that.

Or at least add a comment that would explain what each letter means.

Reply:
UpperCamelCase is the rust convention if no single-letter names are used. I'll change it.